go.mod

```diff
@@ -4,6 +4,7 @@ go 1.17
 
 require (
 	github.com/MakeNowJust/heredoc v1.0.0
+	github.com/Masterminds/sprig v2.22.0+incompatible
```

Comment:
Any specific reason we didnt pick up the newer version of spirg library `v3.2.0`?
`v2.22.0` is more than 2 years old.

The v3 release seems to be having some improvements related to edge cases to their semver functions.

Reply:
@ykakarap nice catch!!